Emergence/Contexts/App Setup/Auth and Slideshow/SlideshowView.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+/// Takes a [String] as imagePaths and goes through them all
+
+class SlideshowView: UIView {
+    var imagePaths: [String] = []
+    private var index = -1
+
+    private var imageView: UIImageView {
+        let imageView = UIImageView(frame: self.bounds)
+        addSubview(imageView)
+        imageView.contentMode = .ScaleAspectFill;
+        imageView.image = self.imageAtIndex(0)
+        return imageView
+    }
+
+    private func imageAtIndex(index: Int) -> UIImage {
```

Comment:
Lots of `!` here, which kind of makes sense, but two other solutions come to mind:

- Create an `enum` for accessing the images instead of using `Int`, and trust that if we change the images, we'll update the `enum`.
- Have this return `UIImage?` and remove the crash operator.

I like the second one personally, since it's easy, and `UIImageView`'s `image` property can take a `nil` anyway, so you shouldn't need any other code changes.